manifest/v1alpha/report/report_type.go

```diff
@@ -0,0 +1,14 @@
+package report
+
+//go:generate ../../../bin/go-enum  --values
+
+// ReportType represents the specific type of report.
```

Comment:
it'd be nice to reference actual objects here, Go docs has a link syntax which would look like this:

```suggestion
// ReportType represents the specific type of [Report].
```